src/xdg-email.c

```diff
@@ -137,7 +122,17 @@ main (int argc, char *argv[])
 
   g_variant_builder_add (&opt_builder,
                          "{sv}",
-                         "address", g_variant_new_string (*uris));
+                         "addresses", g_variant_new_strv ((const char * const *)addresses, -1));
```

Comment:
Mmmh, now I wonder about older hosts. Would it make sense to set both `address` and `addresses` for compatibility? Does the portal handle the case where both are set?

Reply:
yes, that would be preferable. So far, I make the portal merge address and addresses - probably need to check for overlap